test/engine/gh-5958-rollbacked-replace-background-built-index.test.lua

```diff
@@ -0,0 +1,172 @@
+test_run = require('test_run')
+inspector = test_run.new()
+fiber = require('fiber')
+
+box.cfg{}
+
+s = box.schema.space.create('test', {engine = 'memtx'})
+i = s:create_index('pk',  {type='tree', parts={{1, 'uint'}}})
+
+for i=1,5000 do s:replace{i, i, "valid"} end
+started = false
+
+inspector:cmd("setopt delimiter ';'");
+
+function joinable(fib)
+    fib:set_joinable(true)
+    return fib
+end;
+
+function disturb()
```

Comment:
It would be nice if you placed test explanation here i.e. what situation we are trying to simulate and what goals to achive.

Reply:
I'm asking 'cause original test case in the issue seems to be much more smaller...